soroban-env-common/env.json

```diff
@@ -1075,6 +1075,34 @@
                     ],
                     "return": "Void",
                     "docs": "Bumps the expiration ledger of the key specified so the entry will live for `min` ledgers from now. If the current expiration ledger is already large enough to live at least `min` more ledgers, then nothing happens."
+                },
+                {
+                    "export": "8",
+                    "name": "get_contract_id",
```

Comment:
nit: Maybe these should be called `compute` instead of `get`? `get` usually assumes retrieving something that already exists.

Reply:
Sounds good, changed it to `compute`. I agree that makes more sense.

Reply:
Fwiw how the function derives its value is an implementation detail, not part of its interface.

Reply:
To me the difference is that `get` implies that the contract id exists somewhere, which isn't true. Maybe that's just my comprehension though.